server/views/pages/support/offenderDetail.njk

```diff
@@ -143,6 +143,48 @@
         </div>
     </div>
 
+    <details class="govuk-details" data-module="govuk-details">
+      <summary class="govuk-details__summary">
+        <span class="govuk-details__summary-text">
+          View dates stored by Create and vary a licence
+        </span>
+      </summary>
+      <div class="govuk-details__text">
+<table class="govuk-table">
+                <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        <th scope="row" class="govuk-table__header">CRD</th>
+                        <td class="govuk-table__cell">{{ license.licencecrn }}</td>
```

Comment:
This is using CRN, which is an identifier from delius, not a date
It should be using CRD